classes/DirtEventTypes.sc

```diff
@@ -42,7 +42,7 @@ DirtEventTypes {
 				var freq, lag, sustain;
 				var args, midiout, hasGate, midicmd, latency, chan;
 				var sendNRPN, schedmidi, schedmidicmd, donecmd;
-				var hasNote = not (~n == \none and: {~note == \none});
+				var hasNote = (~n != \none or: {~note != nil});
```

Comment:
great! While you are at it, `notNil` is a bit faster, and more idiomatic.

you can check:
```supercollider
{ |x| x != nil }.def.dumpByteCodes
{ |x| x.notNil }.def.dumpByteCodes
```

so:

```supercollider
hasNote = (~n != \none) or: { ~note.notNil };
```

Reply:
Thanks, fixed that!